pkg/actors/actors.go

```diff
@@ -1000,6 +995,10 @@ func (a *actorsRuntime) drainRebalancedActors() {
 	// visit all currently active actors.
 	var wg sync.WaitGroup
 
+	a.lock.Lock()
+	a.internalReminderInProgress = make(map[string]struct{})
```

Comment:
Why is this needed? Don't we have single thread guarantees anymore when using scheduler for reminders?

Reply:
If an internal (workflow) actor reminder execution takes longer than 1m to process, then a duplicate reminder will be executed. To prevent duplication, we need to track what internal actor reminders are in progress to check against.

Existing application reminder guarantees should remain the same

Reply:
As per our call, this code will "ensure" that there is not a memory leak. This code is executed for all actors (internal and external) but is not populating the map. I am fine with this for 1.14 since we plan to revisit internal actors' implementation in 1.15.